app/services/twilio_service.rb

```diff
@@ -31,11 +31,10 @@ def send_unread_message_reminder(url)
   # options[:to] = "+85261111111"
   # options[:body] = "SMS body"
   def send_sms(options)
-    to = @user.mobile.presence || @user.email.presence
-    options = {to: to}.merge(options)
+    options = {to: @user.mobile}.merge(options)
```

Comment:
Great, yes that's fine. might be worth failing if no 'to' field as it's not valid to continue.
What you've implement is more along the lines of defensive programming - preventing an error from occurring but in this case, i think we do want the program to crash so we can find the bug quickly as there should never be a case of sending SMS to a user with no mobile.

```suggestion
    options = {to: @user.mobile}.merge(options)
    raise SomeError.new("User mobile missing") unless options[:to].present?
    
    if send_to_twilio?
    ...
    else
      send to slack
    end
```

Reply:
just a thought

Reply:
Thanks for review. Let me think on this. :)

Reply:
Yes @steveyken. It's good idea to send error message to rollbar. I will add it. Thanks.